packages/template-retail-react-app/app/hooks/use-auth-modal.test.js

```diff
@@ -26,75 +34,18 @@ const mockRegisteredCustomer = {
 }
 
 const mockLogin = jest.fn()
-jest.useFakeTimers()
 
 jest.mock('../commerce-api/auth', () => {
     return jest.fn().mockImplementation(() => {
         return {
-            login: mockLogin.mockImplementation(async () => {
-                throw new Error('invalid credentials')
-            }),
+            login: mockLogin,
             getLoggedInToken: jest.fn().mockImplementation(async () => {
                 return {customer_id: 'mockcustomerid'}
             })
         }
     })
 })
 
-jest.mock('commerce-sdk-isomorphic', () => {
-    const sdk = jest.requireActual('commerce-sdk-isomorphic')
-    return {
-        ...sdk,
-        ShopperLogin: class ShopperLoginMock extends sdk.ShopperLogin {
-            async getAccessToken() {
-                return {
-                    access_token: 'accesstoken',
-                    refresh_token: 'refreshtoken',
-                    customer_id: 'customerId'
-                }
-            }
-            authenticateCustomer() {
-                return {url: '/callback'}
-            }
-        },
-        ShopperCustomers: class ShopperCustomersMock extends sdk.ShopperCustomers {
-            async registerCustomer() {
-                return mockRegisteredCustomer
-            }
-            async getCustomer(args) {
-                if (args.parameters.customerId === 'customerid') {
-                    return {
-                        authType: 'guest',
-                        customerId: 'customerid'
-                    }
-                }
-                return mockRegisteredCustomer
-            }
-            async authorizeCustomer() {
-                return {
-                    headers: {
-                        get(key) {
-                            return {authorization: 'guestToken'}[key]
-                        }
-                    },
-                    json: async () => ({
-                        authType: 'guest',
-                        customerId: 'customerid'
-                    })
-                }
-            }
-            async getResetPasswordToken() {
-                return {
-                    email: 'foo@test.com',
-                    expiresInMinutes: 10,
-                    login: 'foo@test.com',
-                    resetToken: 'testresettoken'
-                }
-            }
-        }
-    }
-})
-
 jest.mock('../commerce-api/utils', () => {
```

Comment:
Did you forget to remove this?

Reply:
It looks like it's in the red (being removed) diff, not sure what you mean?